testcases/misc_tests/cca_export_import_test.c

```diff
@@ -11,14 +11,16 @@
  * by Harald Freudenberger <freude@de.ibm.com>
  */
 
+#if !defined(_AIX)
```

Comment:
You are ifdefing out the `#define _GNU_SOURCE` at many places in your commits. Does it really matter if this is defined on AIX ? Does it cause errors if it is defined? If it is just ignored on AIX, then maybe we better just keep it instead of ifdefing it out at hundreds of places.

Reply:
I actually wanted to add that to all the sources with `AC_DEFINE` instead, but got busy with other stuff. I'll add that in with another commit.